### PR TITLE
Add Cordova

### PR DESCRIPTION
**Pull request by ath0mas**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Cordova is a famous open-source hybrid mobile application development framework.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/cordova/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+aliases: apache-cordova
+created_by: Apache Software Foundation
+display_name: Cordova
+github_url: https://github.com/apache?utf8=%E2%9C%93&q=cordova-
```
Comment (kenyonj) [inline topics/cordova/index.md:5]: seems like this might be a better link?
```suggestion
github_url: https://github.com/apache/cordova
```

**Review by kenyonj — CHANGES_REQUESTED** [review]
This is great. Thank you for the contribution. Just have 1 small suggestion for a better url. Once that's fixed we can get this merged in!

File: topics/cordova/index.md (hunk 1 above)
Comment (ath0mas) [reply topics/cordova/index.md:5]: Right 😃 I followed the _Source Code_ link from Cordova homepage, but this one is best suited

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Pull request merged**